spec/models/card_spec.rb

```diff
@@ -5,59 +5,147 @@
   it { should belong_to :user }
 
   let!(:user) { create(:user) }
+  context 'checking attributes and review' do
 
-  before do
-    @card = user.cards.create(original_text: 'hello', translated_text: 'Привет', review_date: DateTime.current.to_date)
-  end
-  describe 'card has valid attributes' do
 
-    it 'validates card has  user_id' do
-      expect(@card.user.id).to eq user.id
+    before do
+      @card = user.cards.create(original_text: 'hello', translated_text: 'Привет',
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.